src/arvcamera.c

```diff
@@ -2427,6 +2427,65 @@ arv_camera_is_binning_available (ArvCamera *camera, GError **error)
 	return horizontal && vertical;
 }
 
+/**
+ * arv_camera_get_representation_gain:
+ * @camera: a #ArvCamera
+ * @error: a #GError placeholder, %NULL to ignore
+ *
+ * Return: enum ArvGcRepresentation, -1 if not available.
+ *
+ * Since: 
+ */
+
+ArvGcRepresentation
+arv_camera_get_representation_gain (ArvCamera *camera, GError **error)
+{
+	ArvCameraPrivate *priv = arv_camera_get_instance_private (camera);
+
+	g_return_val_if_fail (ARV_IS_CAMERA (camera), FALSE);
```

Comment:
Type mismatch, return REPRESENTATION_UNDEFINED instead, and update the doc accordingly

Reply:
Updated in 17a2454